packages/console/src/common/ConsoleUtils.ts

```diff
@@ -55,20 +52,6 @@ class ConsoleUtils {
     return `${hours}:${minutes}:${seconds}.${milliseconds}`;
   }
 
-  static defaultHost(): string {
-    let defaultHost = window.location.hostname;
-    const apiUrl = process.env.REACT_APP_CORE_API_URL;
-    if (apiUrl != null) {
-      try {
-        const url = new URL(apiUrl);
-        defaultHost = url.hostname;
-      } catch (error: unknown) {
-        log.error('API_URL failed', error);
-      }
-    }
-    return defaultHost;
-  }
-
```

Comment:
I couldn't find any uses of this in DHC or DHE

Reply:
Good to delete then!